install/install.sh

```diff
@@ -93,10 +101,28 @@ echo "Checking for docker."
 ## We unset this variable for this command to avoid conflicts with companion version
 docker --version || curl -fsSL https://get.docker.com | env -u VERSION sh || (
     echo "Failed to start docker, something is wrong."
-    echo "Please report this problem."
-    exit 1
+    echo "Are in in CI? trying dind."
 )
 systemctl enable docker
+if [ $RUNNING_IN_CI -eq 1 ]
+then
+
+
+set -x && \
```

Comment:
A comment explaining what is going on may help (what is dind, who is dind, why dind is important, what dind is doing and etc).